src/NzbDrone.Core/DecisionEngine/Specifications/RssSync/AvailabilitySpecification.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using System.Linq;
+using NLog;
+using NzbDrone.Core.IndexerSearch.Definitions;
+using NzbDrone.Core.Parser.Model;
+using NzbDrone.Core.Tv;
+
+namespace NzbDrone.Core.DecisionEngine.Specifications.RssSync
+{
+    public class AvailabilitySpecification : IDecisionEngineSpecification
+    {
+        private readonly Logger _logger;
+
+        public AvailabilitySpecification(Logger logger)
+        {
+            _logger = logger;
+        }
+
+        public RejectionType Type => RejectionType.Permanent;
+
+        public Decision IsSatisfiedBy(RemoteMovie subject, SearchCriteriaBase searchCriteria)
+        {
+            if (searchCriteria != null)
+            {
+                if (searchCriteria.UserInvokedSearch)
+                {
+                    _logger.Debug("Skipping availability check during search");
+                    return Decision.Accept();
+                }
+            }
+
+            if (subject.Movie.Status != MovieStatusType.Released)
```

Comment:
@galli-leo : like what if we just made this check definable by the user on the object... basically a field in addition to monitor/unmonitor that the user can specify... independent of it.. (but sort of related) with a default value.: 

Announced (basically just skip this check)
Cinematic release (For grabbing cams, ts, etc.)
Bluray / web release (for grabbing blurays, web-dl)
predb: When a scene release of the movie appears on PreDB, monitor the movie and download accordingly (Useful for grabbing Screeners, early blurays etc.) Will probably be the default.